test/unit/video.test.js

```diff
@@ -59,6 +59,15 @@ test('should add the value to the languages object with lower case lang code', f
   deepEqual(result, globalOptions['languages'][code.toLowerCase()], 'should also match');
 });
 
+test('should allow setting global options', function() {
+  videojs.setGlobalOptions({
```

Comment:
The fact this does a merge and not an overwrite strikes me as really odd but that's a separate discussion.